Cargo.toml

```diff
@@ -8,8 +8,12 @@ license = "MIT"
 name = "ordered-multimap"
 readme = "README.md"
 repository = "https://github.com/sgodwincs/ordered-multimap-rs"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
I don't think adding a new feature is a breaking change? I think this can be 0.3.1

Reply:
Yes, will change it.